ios/FluentUI/Shimmer/ShimmerView.swift

```diff
@@ -24,12 +24,12 @@ public enum ShimmerStyle: Int, CaseIterable {
         }
     }
 
-    var defaultTintColor: UIColor {
+    func defaultTintColor(view: UIView) -> UIColor {
```

Comment:
Why do you need a `view` argument here? Can you just access `fluentTheme` directly below?

Reply:
I can only access `fluentTheme` from a UIView and this color is defined in one,.

Reply:
Oh I see, and this control isn't tokenized yet in this branch. In that case, could you change the API to take the `fluentTheme` instead of the view to minimize the dependency between the two?

Reply:
Makes sense, I'll make the change!